tools/codegen/generate.bzl

```diff
@@ -19,7 +19,7 @@ MIXER_IMPORT_MAP = {
 # including the "../.." is an ugly workaround for differing exec ctx for bazel rules
 # depending on whether or not we are building within mixer proper or in a third-party repo
 # that depends on mixer proper. 
-MIXER_IMPORTS = [ "external/com_github_istio_api", "../../external/com_github_istio_api", "external/com_github_istio_mixer" ]
+MIXER_IMPORTS = [ "external/com_github_istio_api", "../../external/com_github_istio_api", "external/com_github_istio_mixer" ] # [, "bazel-out/darwin_x86_64-fastbuild/genfiles/" ]
```

Comment:
Do we need the comment ?

Reply:
no. we do not. when this is ready to submit, i'll trim it off. i needed it to test a hypothesis earlier.